qemu/tests/cfg/boot_guest_with_viommu.cfg

```diff
@@ -0,0 +1,16 @@
+- boot_guest_with_viommu:
+    virt_test_type = qemu
+    type = boot
+    required_qemu= [7.0.0,)
+    only q35
+    only Linux
+    only virtio_net
+    serial_login = no
+    login_timeout = 240
+    virtio_iommu = yes
+    nic_extra_params = ",iommu_platform=on"
+    variants:
+        - direct_plug:
+              virtio_iommu_direct_plug = yes
+        - non_direct_plug:
```

Comment:
This is not a useful scenario.

Reply:
Ok, thanks for sharing this info. It will be changed in the next update.